src/scsi.c

```diff
@@ -131,6 +131,30 @@ int scsi_ses_write(struct block_device *device, enum ibpi_pattern ibpi)
 	return ses_write_msg(ibpi, &device->enclosure->ses_pages, device->encl_index);
 }
 
+int scsi_ses_write_enclosure(struct enclosure_device *enclosure, int idx, enum ibpi_pattern ibpi)
```

Comment:
Will be great if you can an status enum for scsi but as I said before it is optional.